AMEBI.Domain/EF/Context.cs

```diff
@@ -0,0 +1,22 @@
+using AMEBI.Domain.Model;
+using Microsoft.EntityFrameworkCore;
+
+namespace AMEBI.Domain.EF
+{
+    public class Context : DbContext
```

Comment:
Consider renaming Context class to something more verbose. At first glance, I wouldn't notice it's related to database operations.